src/themes/default.css

```diff
@@ -122,16 +128,17 @@
   --bl-font-title-4-semibold: var(--bl-font-weight-semibold) var(--bl-font-title-4);
   --bl-font-title-4-bold: var(--bl-font-weight-bold) var(--bl-font-title-4);
 
-  /* Font Style: Form Label */
-  --bl-font-form-label: var(--bl-font-weight-medium) var(--bl-font-size-xs) var(--bl-font-family);
```

Comment:
You remove form-label but currently input and select labels are using this variable. So they are broken now. Maybe you missed to update them?

Reply:
yes, you're right I fixed them 👍